src/main/java/org/testng/internal/Parameters.java

```diff
@@ -102,7 +102,7 @@
   public static Object[] createConfigurationParameters(Method m,
       Map<String, String> params,
       Object[] parameterValues,
-      @Nullable ITestNGMethod currentTestMethod,
```

Comment:
Done. I reworked the commit:  https://github.com/cbeust/testng/pull/1370/commits/6de2a49c5321dfb1e5aeaaf36578e8f6c577713e